modules/gitops-openshift-go-common-terms.adoc

```diff
@@ -0,0 +1,144 @@
+A custom Kubernetes controller that exists within Argo CD and adds support for an `ApplicationSet` CRD.  This controller automatically generates Argo CD applications based on the contents of an `ApplicationSet` CR and manages them across multiple clusters declaratively from a single mono-repository.
+
+AppProject::
+A CRD representing a logical grouping of applications within a project that governs where and how an application might manage resources. In addition, as an administrator, you use the `AppProject` CRD to restrict where and how non-admin Argo CD users might access those applications.
+
+Argo CD API server::
+A gRPC/REST server that exposes the API consumed by the web UI, CLI, continuous integration (CI), and continuous deployment (CD) systems. 
+
+Argo CD::
+An open-source declarative tool that automates the continuous deployment of Kubernetes-based infrastructure and applications across clusters and development lifecycles.
+
+Argo CD application::
+An application that tracks the continuous deployment of individual Kubernetes resources from the {gitops-shortname} repository, where the resources are defined as manifests, to a target Kubernetes cluster.
+
+`ArgoCD` CRD::
+A Kubernetes CRD that describes the wanted state for a given Argo CD cluster that allows you to configure the components that make up an Argo CD cluster.
+
+Argo CD instance::
+A single installation of Argo CD within a namespace, encapsulating all of the stateful aspects of a running Argo CD.
+
+Argo CD project::
+An entity within Argo CD that refers to the Argo CD open source project's link:https://argo-cd.readthedocs.io/en/stable/user-guide/projects[specific concept of projects], and the corresponding link:https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#projects[`AppProject` CR].
++
+You might deploy Argo CD applications in an Argo CD project across different namespaces or clusters. In contrast, an link:https://docs.openshift.com/container-platform/latest/getting_started/openshift-overview.html#getting-started-openshift-common-terms_openshift-overview[OpenShift project] is restricted to a single namespace and is equivalent in concept to a namespace.
++
+Argo CD project controls the behavior of Argo CD by restricting access to Git repository and remote clusters. Examples include using Argo CD project to control users by restricting who can access certain Argo CD applications or cluster resources through Argo CD UI or Argo CD CLI.
+
+Argo CD repository server (Argo CD-repo-server)::
+An Argo CD component that reads from Git repositories, generates corresponding application manifests, runs custom configuration management tooling, and returns the result to the Argo CD Application Controller.
+
+Argo CD resource (`ArgoCD` CR)::
+A CR that describes the wanted state for a given Argo CD instance that allows you to configure the components and settings which make up that Argo CD instance.
+
+Argo CD server (Argo CD-server)::
```

Comment:
Hasn't this been mentioned above already?

Reply:
I do not see it anywhere. If you see it, please help me locate it. Thank you!